pkg/webui/console/views/admin-packet-broker/default-gateway-visibility.js

```diff
@@ -55,18 +59,20 @@ const DefaultGatewayVisibilityView = () => {
   )
 
   return (
-    <Col md={12}>
-      <Message
-        content={m.gatewayVisibilityInformation}
-        component="p"
-        className={style.routingPolicyInformation}
-      />
-      <GatewayVisibilityForm
-        onSubmit={handleDefaultGatewayVisibilitySubmit}
-        initialValues={initialValues}
-        error={formError}
-      />
-    </Col>
+    <RequireRequest requestAction={getHomeNetworkDefaultGatewayVisibility()}>
```

Comment:
So when this component renders, the home network default gateway visibility should already have been fetched in `admin-packet-broker.js`. Any idea why this is not sufficient?

Reply:
After the refactor, it is not being fetched there anymore

Reply:
Ah I see. In that case, I think it's actually better to move the request to `default-gateway-visibility.js` as you had it before.